### ISSUE TITLE
Mcinabox crash 1.1.4 low fps

### ISSUE THREAD

**Issue by minerchino** [AOF-Dev/MCinaBox#430]
When I play mcinabox 0.1.4 crash above 1.8 with no forge can you fix it in the full release because my phone isn't that good redmi note 7 is my phone I'm thinking about buying a new phone can you recommend a phone that can run 1.12.2 forge? 
And when I'm playing in version 1.1.4 I have low fps in 1.8 and if in version 1.1.3 I have 43 fps? And in 1.7.10 when I'm playing in 1.1.3 I get 90 fps and if in 1.1.4 I get 20 fps? Can you fix that

**Comment by foenerich** [comment]
I also had this problem. I installed forge + optifine and turned all the setting down to 0 or as low as possible. Sure the game now isn't loading with 300FPS but it's enough to enjoy the game. Optifine also brings multiple settings for turning down the graphic settings. I hope this helps you if not just write it in the post.

**Comment by Kotarou2k4** [comment]
I use Redmi note 7 too and i got about 80fps on 1.12.2 play with 4 chunks

**Comment by minerchino** [comment]
Kotarou2k4 How do you play 1.12.2 on your phone is it forge??
Foenerich how I still have 20 and it's annoying I'm thinking about buying a new phone because this phone has problems like hanging when getting opened

**Comment by Kotarou2k4** [comment]
Oh i forgot to say that i use optifine

**Comment by longjunyu2** [comment]
@minerchino I do not agree with all of your comments.
0.1.4 with aarch64 runtime allows devices to allocate more memory.
This will improve the performance in large-memory devices.

If you still think the performance is worse than 0.1.3, then you can continue to use 0.1.3. I will do some tests to find if the performance improved... and in what condition it will be improved.

**Comment by minerchino** [comment]
Longjunyu2 but can aarch64 work on 1.1.3 I'm just. Curious

**Comment by minerchino** [comment]
@longjunyu2 will this work for mcinabox 1.12.2
14nm Octa-Core Snapdragon 660 SoC

**Comment by Kotarou2k4** [comment]
@minerchino redmi note 7 can use aarch64 runtime pack, and I recommend you to play with optifine mod that will improve a lot of performance

**Comment by longjunyu2** [comment]
@minerchino No. aarch64 runtime can not be used on 0.1.3, because.0.1.3 does not have a RuntimeManager to fit runtime of different platform.
